### PR TITLE
add max & default values for standard "limit" query param description

### PR DESCRIPTION
**Pull request by suremarc**
See polygon-io/frontend#114. Currently, the News V2 docs don't tell the user about the default and max values for `limit`. 

I've also refactored the spec for Tickers V3 to reference the pagination spec. Both support the `next_url` pagination parameter, so it seems logical to add that to the description for both.

### CODE REVIEW

**Comment by jbonzo** [comment]
@suremarc Could you post screenshots for Tickers V3 showing before an after to prove no change. I can show you how to get ui-docs to use local spec

File: _specs_/rest/pagination.yml [hunk 1]
```diff
@@ -2,7 +2,10 @@ LimitQueryParam:
   name: limit
   in: query
   description: |
-    Limit the maximum number of results returned.
+    Limit the size of the response, default is 100 and max is 1000.
+
+    If your query returns more than the max limit and you want to retrieve the next page of results,
+    see the `next_url` response attribute.
```
Comment (jbonzo) [inline _specs_/rest/pagination.yml:8]: Do we want these to be the same for all APIs though? I feel like each API would have their own settings specified. @calebcase thoughts?

**Comment by suremarc** [comment]
> @suremarc Could you post screenshots for Tickers V3 showing before an after to prove no change. I can show you how to get ui-docs to use local spec
Before:
before
After:
after

Note the addition of the `default` parameter. Actually, it might make sense to remove `default, `maximum`, and `example` values from the description, since it appears those values are in the spec as their own fields. Maybe not in this PR, though, since I think the FE team would have to change our UI to use those values. 

Also note that `next_page_path` is now `next_url`, which is the correct response attribute if I'm not mistaken.

**Comment by jbonzo** [comment]
> > @suremarc Could you post screenshots for Tickers V3 showing before an after to prove no change. I can show you how to get ui-docs to use local spec
> > Before:
> 
> before
> After:
> 
> after
> 
> Note the addition of the `default` parameter. Actually, it might make sense to remove `default, `maximum`, and `example` values from the description, since it appears those values are in the spec as their own fields. Maybe not in this PR, though, since I think the FE team would have to change our UI to use those values.
> 
> Also note that `next_page_path` is now `next_url`, which is the correct response attribute if I'm not mistaken.

@suremarc I was looking for screenshots of our UI, that looks like swagger default

File: _specs_/rest/pagination.yml (hunk 1 above)
Comment (calebcase) [reply _specs_/rest/pagination.yml:8]: If we can come up with a general default and max it seems reasonable to put that here. The individual endpoints should still be able to override them vie the `$ref + more` trick.

**Comment by suremarc** [comment]
Tickers V3:

Before:
image

After: 
image

**Comment by suremarc** [comment]
Ticker News V2:

Before:
image

After:
image

**Review by jbonzo — APPROVED** [review]

**Pull request merged**